config/initializers/assets.rb

```diff
@@ -8,7 +8,7 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w(misc/*.js email.css ie*.css)
+Rails.application.config.assets.precompile += %w(app/controllers/*.js misc/*.js email.css ie*.css)
```

Comment:
I just added the file the the `misc` dir to get this moving along. If we want to address restructuring JS we can do it in a later PR!